jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -250,8 +253,18 @@ protected void onContinue() {
 	}
 
 	public boolean isDone() {
```

Comment:
Why not just do it in java? We'll need to mirror the bit definitions, but it's hardly the first time. Since this is only a read, no atomic/sync is required.